packages/@functionless/ast/package.json

```diff
@@ -5,6 +5,7 @@
   "types": "lib/index.d.ts",
   "scripts": {
     "build": "tsc -b",
+    "dev": "tsc --watch",
```

Comment:
We no longer need this in each of the child packages. We can instead run a single `tsc -w` from the root

Reply:
Shouldn't we be building the same way locally that it will build in ci?

Reply:
Not sure it would make a difference? If the references are set up properly, tsc -w would be optimal performance and identical behavior.